examples/gin/main.go

```diff
@@ -0,0 +1,136 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/airbrake/gobrake/v5"
+	ginbrake "github.com/airbrake/gobrake/v5/gin"
+	"github.com/gin-gonic/gin"
+)
+
+var ProjectId int64 = 105138                               // Replace it with your project Id
+var ProjectKey string = "fd04e13d806a90f96614ad8e529b2822" // Replace it with your project key
+
+// Define models
+
+type Date struct {
+	Date int64 // the server date
+}
+
+type WeatherInfo struct {
+	Lat            float32       `json:"lat,omitempty"`
+	Lon            float32       `json:"lon,omitempty"`
+	Timezone       string        `json:"timezone,omitempty"`
+	TimezoneOffset float32       `json:"timezone_offset,omitempty"`
+	Current        interface{}   `json:"current,omitempty"`
+	Minutely       []interface{} `json:"minutely,omitempty"`
+	Hourly         []interface{} `json:"hourly,omitempty"`
+	Daily          []interface{} `json:"daily,omitempty"`
+}
+
+var availableLocations = []string{
+	"austin",
+	"pune",
+	"santabarbara",
+}
+
+var notifier = gobrake.NewNotifierWithOptions(&gobrake.NotifierOptions{
+	ProjectId:   ProjectId,    // use it as an environment variable
```

Comment:
I don't understand the comment. On line 18 in this file we recommend replacing it with our own values and here we recommend using an environment variable. I think we should stick to env variables and this file should support them.

Reply:
I am removing the comment to make it like the example in apexlog.